identify/extensions.py

```diff
@@ -246,18 +253,23 @@
     'BUILD': {'text', 'bazel'},
     'BUILD.bazel': {'text', 'bazel'},
     'CMakeLists.txt': EXTENSIONS['cmake'],
+    'CHANGELOG': EXTENSIONS['txt'],
+    'CONTRIBUTING': EXTENSIONS['txt'],
     'COPYING': EXTENSIONS['txt'],
+    'COPYING.LESSER': EXTENSIONS['txt'],
```

Comment:
iirc this isn't necessary -- it'll get covered by `COPYING`

Reply:
Confirmed, thanks. Dropped that line.